thunderfish/fishfinder.py

```diff
@@ -1,878 +1,878 @@
-#!/usr/bin/python

```

Comment:
I did not change anything, but github marks it as completely changed. fishfinder.py is still working. I consider this a bug and would just ignore it.